state_publisher_node:
  ros__parameters:
    odom_topic: /ego_racecar/odom
    drive_topic: /drive
    lidar_topic: /scan
    state_obs_topic: /ego_racecar/states
    local_frame: ego_racecar/base_link
    global_frame: map